.travis.yml

```diff
@@ -33,7 +33,7 @@ python:
 
 env:
     global:
-        - COLUMNS=240
+        - COLUMNS=180
```

Comment:
what this

Reply:
Determines the width of the output in travis.